docs/yaml/functions/_build_target_base.yaml

```diff
@@ -228,13 +228,15 @@ kwargs:
       Set this to `[]`, or omit the keyword argument for the default behaviour.
 
   override_options:
-    type: list[str]
+    type: list[str] | dict[str | bool | int | list[str]]
     since: 0.40.0
     description: |
       takes an array of strings in the same format as `project`'s `default_options`
       overriding the values of these options
       for this target only.
 
+      *Since 1.2.0* a dictionary may now be passed
```

Comment:
Seems like other places in this PR use a different style for this comment.

`*(since 0.62.0)* Dictionary order is guaranteed to be insertion order.`

Note the capital letter to start and the ending period along with the styling of since.

Reply:
Same comment in other sections

Reply:
It would be nice to be able to make this more standard through refman.

```yaml
  override_options:
    type: list[str] | dict[str | bool | int | list[str]]
    since: 0.40.0
    changes:
      - since: 1.2.0
        description: A dictionary may now be passed.
```

Remove some of the manual stuff.

Reply:
I agree it would be nice to have those "since" part of refman, but I don't think we have that currently, do we?

Changed the format to be more consistent.